.github/workflows/build.yml

```diff
@@ -76,3 +76,4 @@ jobs:
         shell: bash
         run: |
           west twister -s sample.net.wifi -p siwx917_rb4338a -v --inline-logs -K
+          west twister -s sample.net.wifi -p siwx917_rb4338a -x EXTRA_CONF_FILE=$(pwd)/samples/net/wifi/shell/soc/siwx917_offloaded_sockets.conf -v --inline-logs -K
```

Comment:
Hmm, except it seems the test *is* in the same tree in this case.

Reply:
`sample.net.wifi` come from upstream. Only `siwx917_offloaded_sockets.conf` is located in the downstream.

Reply:
> I suppose the clean way to do this would be to define a new test case in `sample.yaml`

Yes, I expect to do that when support for 917 will be merged.